azure-iot-device/tests/common/test_mqtt_transport.py

```diff
@@ -400,24 +400,49 @@ def test_calls_paho_connect(
             host=fake_hostname, port=port, keepalive=mocker.ANY
         )
 
-    @pytest.mark.it("Passes DEFAULT_KEEPALIVE to paho connect function")
+    @pytest.mark.it("Passes keep alive to paho connect function")
     @pytest.mark.parametrize(
```

Comment:
Actually, now that we have dynamic keepalive values, we can fold this test into the one above with a simple addition of the keep_alive parametrization (and an adjustment to the check on the keepalive call value on line 400)